debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -505,7 +504,8 @@ protected boolean isGeometryCollection(String upperCaseTypeName) {
 
     protected List<String> extractEnumAndSetOptions(Column column) {
 //        return MySqlDdlParser.parseSetAndEnumOptions(column.typeExpression());
-        return MySqlAntlrDdlParser.parseSetAndEnumOptions(column.typeExpression());
+//        return MySqlAntlrDdlParser.parseSetAndEnumOptions(column.typeExpression());
```

Comment:
Is that method `parseSetAndEnumOptions()` then even needed any longer? Also the commented lines should be removed alltogether.